src/main/java/uk/gov/pay/api/model/PaymentWithAllLinks.java

```diff
@@ -13,10 +13,10 @@
     private PaymentLinks links = new PaymentLinks();
 
     private PaymentWithAllLinks(String chargeId, long amount, PaymentState state, String returnUrl, String description,
-                                String reference, String paymentProvider, String createdDate, RefundSummary refundSummary,
-                                List<PaymentConnectorResponseLink> paymentConnectorResponseLinks,
+                                String reference, String email, String paymentProvider, String createdDate,
```

Comment:
Minor: This one has grown to 15 arguments as well, a refactoring task later perhaps